code/__DEFINES/role_preferences.dm

```diff
@@ -23,7 +23,7 @@
 #define ROLE_DEVIL				"devil"
```

Comment:
We'll need to change this at some point, it works for now. Though.

Reply:
why do you need to change it at some point

Reply:
@Qustinnus Because borg will need to be able to be spawned in gamemodes other than Borg mode.